src/main/java/medicaldoctor/controllers/DoctorDashboardController.java

```diff
@@ -11,10 +17,22 @@
 public class DoctorDashboardController implements Initializable, ParentController{
     ControllerManager cm;
     
+    @FXML
+    private TableView<DoctorTable> table;
+   
+    @FXML
+    private TableColumn<DoctorTable, String> name, doctor, date;
+    
+    private ObservableList<DoctorTable> list = FXCollections.observableArrayList(
+        //Ask Thomas about this
+        //new DoctorTable();
+    );
+    
     @Override
     public void initialize(URL url, ResourceBundle rb) {
-        
+        table.getColumns().addAll(name,doctor,date);
```

Comment:
this shouldn't be in this pull request, but i dont think its going to break anything so its fine

Reply:
this meaning all of this in this file